_inc/lib/core-api/wpcom-endpoints/publicize-connections.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+/**
+ * Publicize: List Connections
+ *
+ * [
+ *   { # Connnection Object. See schema for more detail.
+ *     id:           (string)  Connection unique_id
+ *     service_name: (string)  Service slug
+ *     display_name: (string)  User name/display name of user/connection on Service
+ *     global:       (boolean) Is the Connection available to all users of the site?
+ *   },
+ *   ...
+ * ]
+ * @since 6.8
+ */
+class WPCOM_REST_API_V2_Endpoint_List_Publicize_Connections extends WP_REST_Controller {
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'publicize/connections';
+
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Called automatically on `rest_api_init()`.
+	 */
+	public function register_routes() {
+		register_rest_route( $this->namespace, '/' . $this->rest_base, array(
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_items' ),
+				'permission_callback' => array( $this, 'get_items_permission_check' ),
+			),
+			'schema' => array( $this, 'get_public_item_schema' ),
+		) );
+	}
+
+	/**
+	 * Helper for generating schema. Used by this endpoint and by the
+	 * Connection Test Result endpoint.
+	 *
+	 * @internal
```

Comment:
Some of these methods could use `@return`

Reply:
ea8627f70